config/settings/production.py

```diff
@@ -0,0 +1,14 @@
+from config.env import env
+
+from .base import *
+
+DEBUG = env("PRODUCTION_DEBUG")
```

Comment:
just have DEBUG if its a production environment it should be False and dev it should be True.